<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QWidget" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="contenedorInicial" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#contenedorInicial {
  background-color: #00273d;
}</string>
   </property>
   <widget class="QWidget" name="logo" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnLogo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/assets/logo1.png</normaloff>:/assets/logo1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>131</width>
       <height>131</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="login" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblCorreo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Correo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnCorreo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblContrasena">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Contraseña: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnContrasena">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogIn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Acceder</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblBienvenida">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¡Bienvenid@!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¿No tienes una cuenta?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistro">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
